consensus/istanbul/backend/backend_test.go

```diff
@@ -184,6 +188,43 @@ func TestGetProposer(t *testing.T) {
 	}
 }
 
+func TestIsQBFTConsensus(t *testing.T) {
+	chain, engine := newBlockChain(1, big.NewInt(2))
+	defer engine.Stop()
+	qbftConsensus := engine.IsQBFTConsensus()
+	if qbftConsensus {
+		t.Errorf("IsQBFTConsensus() should return false")
+	}
+
+	// Set the value of qbftBlock to 1
+	qbftConsensus = engine.IsQBFTConsensus()
```

Comment:
isn't code block duplicated? and the comment seems unrelated

Reply:
Removed duplicate code block